library/src/main/java/com/xwray/groupie/GroupAdapter.java

```diff
@@ -209,6 +209,15 @@ public void remove(@NonNull Group group) {
         groups.remove(position);
         notifyItemRangeRemoved(itemCountBeforeGroup, group.getItemCount());
     }
+    
+    public void removeAt(int index) {
+        Group group = getGroup(index);
```

Comment:
Once we get hold of group, we can call remove(@NonNull Group group) method to avoid repetition.

Reply:
I actually ended up combining the old and new method into a new private method to avoid additional recursion (in one case, you already have the group, in the other case, you already have the index).